config_override.php

```diff
@@ -1,3 +1,23 @@
 <?php
 /***CONFIGURATOR***/
+$sugar_config['theme_settings']['Suite7']['menu'] = '121212';
```

Comment:
Dear Matt


I start do not do it, Will and Ashley informed me,

how to improve my commits.



Jerzy


________________________________
From: Matt Lorimer <notifications@github.com>
Sent: 07 July 2016 15:50:29
To: salesagility/SuiteCRM
Cc: Jerzy Brzezinski; Author
Subject: Re: [salesagility/SuiteCRM] Adding Events Delegates Filter via Subpanel (#1709)


In config_override.php<https://github.com/salesagility/SuiteCRM/pull/1709#discussion_r69920827>:

> @@ -1,3 +1,23 @@
>  <?php
>  /***CONFIGURATOR***/
> +$sugar_config['theme_settings']['Suite7']['menu'] = '121212';


do not commit this

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1709/files/daa22c19c56a4c1de1cf1a62a308172c75944a2c..52106795574d6815ea24610072da1de8ec12a121#r69920827>, or mute the thread<https://github.com/notifications/unsubscribe/AKP8DNDP1p_b0rZS-rSJ-mS9z_UiknNkks5qTRI1gaJpZM4JF9xd>.